addons/website_profile/models/website.py

```diff
@@ -7,4 +7,4 @@
 class Website(models.Model):
     _inherit = 'website'
 
-    karma_profile_min = fields.Integer(string="Minimal karma to see other user's profile", default=150)
+    karma_profile_min = fields.Integer(string="Minimal karma to see other user's profile")
```

Comment:
As it is now configurable, would keep the default value. Anyway people that want a full open profile page can tweak the value directly in the website view :)

Reply:
thanks @tde-banana-odoo ,
updated the PR